app/models/hathi_trust_links.rb

```diff
@@ -67,6 +67,6 @@ def fulltext_available?
   end
 
   def item_target(id)
-    "https://babel.hathitrust.org/cgi/pt?id=#{URI.encode(id)}&view=1up&seq=9"
+    "https://babel.hathitrust.org/cgi/pt?id=#{ERB::Util.url_encode(id)}"
```

Comment:
We decided to dump these extra query args because 1) they are not needed, and 2) in practice, they are not properly encoded so they do not survive the bounce through Shib.